test/html_to_json/Html_to_jsonTest.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package html_to_json;
+
+import java.io.IOException;
+import static jdk.nashorn.internal.objects.Global.getJSON;
+import net.sf.json.JSONSerializer;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.jsoup.Connection;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import org.skyscreamer.jsonassert.JSONAssert;
+
+/**
+ *
+ * @author FireHawk2300
+ */
+public class Html_to_jsonTest {
+    
+    public Html_to_jsonTest() throws IOException {
+        
+        String html = Jsoup.connect("https://github.com/egis/handbook/blob/master/Tech-Stack.md").get().html();
+        
+        Document doc = Jsoup.parse(html);
+        
+    }
+    // Method used to verify JSON format
+    // Returns boolean result
+    public static boolean isValidJSON(String test) {
```

Comment:
Doesn't have to be a _public_ method, you only use it in your test methods inside this class.